scripts/ufm_links/load_links.py

```diff
@@ -0,0 +1,108 @@
+
+    This software product is a proprietary product of Nvidia Corporation and its affiliates
+    (the "Company") and all right, title, and interest in and to the software
+    product, including all associated intellectual property rights, are and
+    shall remain exclusively with the Company.
+
+    This software product is governed by the End User License Agreement
+    provided with the software product.
+
+@author: Nasr Ajaj
+@date:   Jul 17, 2022
+"""
+
+import os
+import platform
+from http import HTTPStatus
+
+try:
+    from utils.utils import Utils
+    from utils.ufm_rest_client import UfmRestClient, HTTPMethods
+    from utils.args_parser import ArgsParser
+    from utils.config_parser import ConfigParser
+    from utils.logger import Logger, LOG_LEVELS
+    from utils.exception_handler import ExceptionHandler
+except ModuleNotFoundError as e:
+    if platform.system() == "Windows":
+        print("Error occurred while importing python modules, "
+              "Please make sure that you exported your repository to PYTHONPATH by running: "
+              f'set PYTHONPATH=%PYTHONPATH%;{os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))}')
+    else:
+        print("Error occurred while importing python modules, "
+              "Please make sure that you exported your repository to PYTHONPATH by running: "
+              f'export PYTHONPATH="${{PYTHONPATH}}:{os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))}"')
+
+
+class UfmLinksConstants:
+
+    LINKS_API_URL = 'resources/links'
+    API_PARAM_CABLE_INFO = "cable_info"
+    API_PARAM_MONITORING_COUNTERS_INFO = "monitoring_counters_info"
+    API_PARAM_SYSTEM = "system"
+
+    args_list = [
+        {
+            "name": f'--{API_PARAM_SYSTEM}',
+            "help": "Option to get all links data related to specific system node"
+        },
+        {
+            "name": f'--{API_PARAM_MONITORING_COUNTERS_INFO}',
+            "help": "Option to show monitoring counters data"
+        },
+        {
+            "name": f'--{API_PARAM_CABLE_INFO}',
+            "help": "Option to show cable information"
+        }
+    ]
+
+
+class UfmLinksConfigParser(ConfigParser):
+    def __init__(self, args):
+        super().__init__(args)
+
+
+class UfmLinksManagement:
+
+    @staticmethod
+    def get_links(show_cable_info, show_counters_info, system=None):
+        url = f'{UfmLinksConstants.LINKS_API_URL}?{UfmLinksConstants.API_PARAM_CABLE_INFO}={show_cable_info}' \
+              f'&{UfmLinksConstants.API_PARAM_MONITORING_COUNTERS_INFO}={show_counters_info}'
+        if system:
+            url = f'{url}&{UfmLinksConstants.API_PARAM_SYSTEM}={system}'
+        response = ufm_rest_client.send_request(url)
+        if response and response.status_code == HTTPStatus.OK:
+            Logger.log_message(response.json())
+        else:
+            Logger.log_message(response, LOG_LEVELS.ERROR)
```

Comment:
the response is object not string, and by the way no need to log the error here as we already have it in:
https://github.com/Mellanox/ufm_sdk_3.0/blob/main/utils/ufm_rest_client.py#L104